index.html

```diff
@@ -0,0 +1,100 @@
+    <body>
+        <main>
+            <div class="main-text">
+                <nav class="nav-container">
+                    <div class="first"><img class="apple" src="images-apple/logo-apple.png" alt="apple" /></div>
+                    <div>
+                        <a href="#">
+                            <p>Store</p>
+                        </a>
+                    </div>
+                    <div>
+                        <a href="#">
+                            <p>Mac</p>
+                        </a>
+                    </div>
+                    <div>
+                        <a href="#">
+                            <p>iPod</p>
+                        </a>
+                    </div>
+                    <div>
+                        <a href="#">
+                            <p>iPhone</p>
+                        </a>
+                    </div>
+                    <div>
+                        <a href="#">
+                            <p>iPad</p>
+                        </a>
+                    </div>
+                    <div>
+                        <a href="#">
+                            <p>iTunes</p>
+                        </a>
+                    </div>
+                    <div>
+                        <a href="#">
+                            <p>Support</p>
+                        </a>
+                    </div>
+                    <div class="last">
+                        <form>
+                            <input type="text" class="search-box" />
+                            <img class="search" src="images-apple/search1.PNG" alt="search" />
+                        </form>
+                    </div>
+                </nav>
+                <div class="text">
+                    <a href="#">
+                        <p class="what">
+                            What will your verse be?
+                        </p>
+                    </a>
+                    <a href="#">
+                        <p class="everyone">
+                            Everyone has something to share. See how people are using <br />
+                            iPad in their unique ways to add to the world's story.
+                        </p>
+                    </a>
+                    <div class="verse">
+                        <a href="#">
+                            <p class="watch">Watch the "Your Verse" film and explore the stories</p>
+                        </a>
+                        <img src="images-apple/play-button-overlay-png.png" alt="play" />
+                    </div>
+                </div>
+            </div>
+        </main>
+        <section>
+            <h4 class="none"> n </h4>
+            <div class="s-container">
+                <div class="image image1"><a href="#"></a></div>
+                <div class="image image2"><a href="#"></a></div>
+                <div class="image image3"><a href="#"></a></div>
+                <div class="image image4"><a href="#"></a></div>
```

Comment:
You should make the images themselves as anchor tags. You can do this by wrapping the images with an anchor tag